web/source/text/kbdInterface.ts

```diff
@@ -844,14 +848,10 @@ namespace com.keyman.text {
      * @return      {boolean}                 True if the test succeeds 
      */       
     ifStore(systemId: number, strValue: string, outputTarget: OutputTarget): boolean {
-      let keyman = com.keyman.singleton;
-
       var result=true;
-      if(systemId == KeyboardInterface.TSS_LAYER) {
-        // How would this be handled in an eventual headless mode?
-        result = (keyman.osk.vkbd.layerId === strValue);
-      } else if(systemId == KeyboardInterface.TSS_PLATFORM) {
-        result = this.platformSystemStore.matches(strValue);
+      let store = this.systemStores[systemId];
```

Comment:
The logic of this function isn't identical but I *think* it works out the same as before.